<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellClass</class>
 <widget class="QDialog" name="SellClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>602</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>销售管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="management.qrc">
    <normaloff>:/Management/Resources/ico.ico</normaloff>:/Management/Resources/ico.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_29">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>163</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>375</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>销售清单查看</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="goodsID_search_sell_CKB">
            <property name="text">
             <string>按商品编号查询：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="goodsID_search_sell_LE">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="employeeID_search_sell_CKB">
            <property name="minimumSize">
             <size>
              <width>119</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>按收银员查询：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="employeeID_search_sell_LE">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="sellTime_search_sell_CKB">
            <property name="minimumSize">
             <size>
              <width>119</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>按时间查询：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="sellTimeStart_search_sell_DTE">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>----</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="sellTimeEnd_search_sell_DTE">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="sellAll_search_sell_CKB">
            <property name="text">
             <string>所有销售情况查询</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="search_search_sell_BTN">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>163</height>
        </size>
       </property>
       <property name="title">
        <string>销售金额统计</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="goodsID_stat_sell_CKB">
            <property name="text">
             <string>按商品编号统计：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="goodsID_stat_sell_LE">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="employeeID_stat_sell_CKB">
            <property name="minimumSize">
             <size>
              <width>119</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>按收银员统计：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="employeeID_stat_sell_LE">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="sellTime_stat_sell_CKB">
            <property name="minimumSize">
             <size>
              <width>119</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>按时间统计：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="sellTimeStart_stat_sell_DTE">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>----</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="sellTimeEnd_stat_sell_DTE">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QCheckBox" name="sellAll_stat_sell_CKB">
            <property name="text">
             <string>按所有收银员统计</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="stat_stat_sell_BTN">
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="sellInfo_depot_TBW">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>商品编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>收银员</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>销售数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>销售金额</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>折扣</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>会员折扣</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>销售时间</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status_sell_LB">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="management.qrc"/>
 </resources>
 <connections/>
</ui>
